### PR TITLE
Fix Issue #859:Disabled menu item on selection

### LINKED ISSUE #476: Items in option menu are not stable
**Summary:** 

If we click on any menu item in Dashboard Activity and come back then menu item is disappeared that was clicked before.

**Steps to reproduce:** 

1. Click on menu item in dashboard activity. There are 4 menu items.
2. Come back to dashboard activity. Now only three menu items are there.

**Expected behavior:** 

Menu items should not change.

**Observed behavior:** 

The clicked menu item disappears after coming back to Dashboard Activity.

**Device and Android version:** 

Yureka with android 5.1.1

### LINKED ISSUE #859: On clicked appearance in create group
**Summary:** 

Selecting on create group after selecting create client or create center does not lead to faded appearance.

**Steps to reproduce:** 

1)run the app
2)create client
3)create group

**Expected behavior:** 

The create group menu option should be faded.

**Observed behavior:** 

create group menu option appears as if not selected.

**Device and Android version:** 

Android 5.0.2,API 21

**Screenshots:** 

I am working on this.
Note:If we select create group at first.it works right for first time but not subsequently.

### PR DESCRIPTION
**Pull request by AkshGautam**
Fix #476 
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `MifosStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by AkshGautam** [comment]
@tarun0 @therajanmaurya  Please review this for issue #859 . I have tried to reduce the function calls and checked it to verify that it's working in all cases like back button press etc.
Please Have look at all the handled cases.

**Comment by tarun0** [comment]
A fix engendering another issue isn't quite a 'fix' :) Two progressbars are being shown simultaneously in the video you attached.

File: mifosng-android/src/main/java/com/mifos/mifosxdroid/online/DashboardActivity.java [hunk 1]
```diff
@@ -63,6 +63,7 @@
     SwitchCompat userStatusToggle;
     private Menu menu;
     private boolean doubleBackToExitPressedOnce = false;
+    private boolean item_client = true, item_center = true, item_group = true;
```
Comment (tarun0) [inline mifosng-android/src/main/java/com/mifos/mifosxdroid/online/DashboardActivity.java]: Follow camelCase naming convention.

**Comment by AkshGautam** [comment]
@tarun0 the two progress bar problem has already been addressed and solved in issue #830 . It's just that I haven't merged the branches in my local repo. I would definitely change the naming convention. Thanks

**Comment by AkshGautam** [comment]
@tarun0 @therajanmaurya I have made the changes. Please ignore the two progress bars since it is already a persisting issue which I have already solved in another PR.

**Pull request merged**